pytrendy/io/plot_pytrendy.py

```
@@ -35,21 +38,39 @@ def plot_pytrendy(df: pd.DataFrame, value_col: str, segments_enhanced: list[dict
         'Noise': 'lightgray',
     }
 
+    accepted_index_types = ['date', 'integer', 'float', 'string']
+    print(f"internal index type {index_type}")
+    if not (index_type in accepted_index_types):
```

Ruff (0.15.20)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)